frontend/tests/features/collection/revision.test.ts

```diff
@@ -142,26 +142,45 @@ describeIfDeployed("Collection Revision", () => {
 });
 
 async function startRevision(): Promise<string> {
-  // (thuang): NOTE: the `*` captures the "collection-row" selector
-  const COLLECTION_ROW_SELECTOR_START = `*${getTestID(
-    COLLECTION_ROW_ID
-  )} >> text="Start Revision"`;
-
   await goToPage(TEST_URL + ROUTES.MY_COLLECTIONS);
-  await page.waitForLoadState();
+  // (thuang): Wait for collections to load to prevent race condition
+  await page.waitForLoadState("networkidle");
+
+  const MIN_USABLE_COLLECTION_COUNT = 4;
```

Comment:
This is arbitrary, originally I was thinking about 5, but generally at least 50% of the seeded published collections need to be available. So if we seed 10 published collections, around 4 to 5 should be available for revision (AKA don't have a revision in progress already)